addons/event/migrations/11.0.1.0/openupgrade_analysis_work.txt

```diff
@@ -0,0 +1,87 @@
+---Fields in module 'event'---
+event        / event.event              / auto_confirm (boolean)        : not a function anymore
+# Nothing to do
+
+event        / event.event              / date_begin_located (datetime) : type is now 'char' ('datetime')
+# Nothing to do
+
+event        / event.event              / date_end_located (datetime)   : type is now 'char' ('datetime')
+# Nothing to do
+
+event        / event.event              / is_online (boolean)           : NEW
+# Nothing to do
+
+event        / event.event              / reply_to (char)               : DEL
```

Comment:
You have to put as follower this email address, searching for a partner with that email or creating a new one if not exist.